Services/AuthentificationService/Controllers/AuthentificationController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AuthentificationService.Controllers
+{
+    [ApiController]
+    public class AuthentificationController : ControllerBase
+    {
+        /// <summary>
+        /// Storage for tokens 
+        /// May be used until database is created
+        /// </summary>
+
+        /// <summary>
+        /// Constructor for lulz and some data to play with
+        /// </summary>
+       /* public AuthentificationController()
+        {
+            tokens = new List<string>();
+        }*/
+
+        /// <summary>
+        /// Generates and returns an active token
+        /// </summary>
+        [Route("[controller]/get")]
+        [HttpGet]
+        public string GetToken()
+        {
+            var date = DateTime.Now;
+            var randomDigit = new Random();
+            var valueForToken1 = randomDigit.Next(-400000000, 400000000);
+            var valueForToken2 = randomDigit.Next(-60000, 60000);
+            var valueForToken3 = randomDigit.Next(-60000, 60000);
+            var valueForToken4 = randomDigit.Next(-200, 200);
+            var byteArrayForGuid = new byte[]
+            {
+                (byte)date.Day,
+                (byte)date.Month,
+                (byte)(date.Year % 100),
+                (byte)(date.Year / 100),
```

Comment:
Why not something like `BitConverter.GetBytes(DateTime.UtcNow.ToBinary())` instead of this all that stuff?